sor/src/test/java/com/bazaarvoice/emodb/table/db/astyanax/TableLifeCycleTest.java

```diff
@@ -275,15 +311,6 @@ public void testDropPurgeDelete()
             DataPurgeDAO usDataPurgeDAO = mock(DataPurgeDAO.class);
             AstyanaxTableDAO usTableDAO = newTableDAO(backingStore, DC_US, usDataCopyDAO, usDataPurgeDAO, fct);
 
-            // The 2nd purge is delayed until full consistency is achieved relative to the time of the drop.
```

Comment:
It was moved because full consistency must now be achieved before the **first** drop. This is because need to ensure consistency before we write the final null's to the megabus